test/aircraft/Pilot/cancelApproachClearance.spec.js

```diff
@@ -23,26 +26,26 @@ ava('.cancelApproachClearance() returns early if #hasApproachClearance is false'
 });
 
 ava('.cancelApproachClearance() sets the correct modes and values in the Mcp', (t) => {
-    const currentAltitudeMock = 7000;
     const nextAltitudeMock = 4000;
     const headingBeforeLocalizerInterceptionMock = 3.839724354387525; // 220 in degrees
     const headingAfterLocalizerInterceptionMock = 4.36332;  // appx 250 degrees
     const nextHeadingDegreesMock = 250;
-    const currentSpeedMock = 210;
     const nextSpeedMock = 180;
     const shouldExpediteDescentMock = false;
     const shouldUseSoftCeilingMock = false;
     const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
+    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, navigationLibraryFixture);
+    const currentAltitudeMock = model.speed;
```

Comment:
This is where the test is failing.

Reply:
putting in an altitude they'd be descending through to get to the assigned altitude would fix the problem. For example, you can set currentAltitudeMock to `5700` and the test will pass.

Reply:
👍

Reply:
i set it to speed

Reply:
why do i do this